print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
```

Comment:
What is the purpose of this constructor? What is value and what is next? Should we have more specific naming here?